CMakeLists.txt

```diff
@@ -2,5 +2,6 @@
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 2.6)
 find_package(Rock)
+add_definitions(-fext-numeric-literals)
```

Comment:
Instead of adding yet-another-compiler-flag (which should be tested before being added btw.), can't we change the code to not use the boost macro for initialization ?